Makefile

```diff
@@ -222,6 +228,26 @@ else
 	cd devworkspace-crds && git checkout $(DEVWORKSPACE_API_VERSION) && git reset --hard origin/$(DEVWORKSPACE_API_VERSION)
 endif
 
+### update_terminal_manifests: pull latest web terminal manifests to web-terminal-operator. Note: pulls master branch by default
+update_terminal_manifests:
+	mkdir -p web-terminal-operator
+	cd web-terminal-operator && git init || true
+ifneq ($(shell git --git-dir=web-terminal-operator/.git remote), origin)
```

Comment:
This line seems to cause all makefile commands to fail for me.

Reply:
seems it's fixed now